faq.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Frequently Asked Questions (FAQ)
+---
+
+- this list will be replaced by the Table of Contents
+{:toc}
+
+## Installation
+
+### How do I install a recent version of Cockpit on Debian or Ubuntu?
+
+While Debian and Ubuntu both ship with access to Cockpit in their main repositories, the version in each might be older than the latest available. This is due to the nature of "stable" and "long term support" distributions.
+
+There's often a newer version of Cockpit available for both Debian and Ubuntu, in an extra "backports" repository. We recommend using this newer version, if you're able to do so.
```

Comment:
Huh, since when did GitHub lose the ability to insert suggestions?

There's often a newer version of Cockpit available in the "backports" repositories, for both [Debian](https://backports.debian.org/) and [Ubuntu](https://wiki.ubuntu.com/UbuntuBackports).

Reply:
```suggestion
There's often a newer version of Cockpit available in the "backports" repositories, for both [Debian](https://backports.debian.org/) and [Ubuntu](https://wiki.ubuntu.com/UbuntuBackports). We recommend using this newer version, if you're able to do so.
```